soil_params_Gupta2020_2022/utils.jl

```diff
@@ -29,26 +31,26 @@ function regrid(data, (lon, lat), resolution, transform, nlayers)
                 (lon .>= lon_min + resolution * (lon_id - 1)) .&
                 (lon .< lon_min + resolution * lon_id)
             x = data[lon_mask, lat_mask, :]
-            x_land_mask = x .!== -3.4f38
+            x_land_mask = .~(typeof.(x) .<: Missing)
```

Comment:
Can you rewrite this so that it is more clear what is happening?

Reply:
I left a comment explaining what this does

Reply:
I think you can write something like
```
x_land_mask = .!ismissing.(x)
```

Reply:
nice, that's cleaner and works.